### ISSUE TITLE
MultipleStatementAlignment sniff also triggers on single assignment without spaces

### ISSUE THREAD

**Issue by dheineman** [squizlabs/PHP_CodeSniffer#2223]
With the `Generic.Formatting.MultipleStatementAlignment` enabled and using the example code below phpcs issues a warning with the message `Equals sign not aligned correctly; expected 1 space but found 0 spaces`.

```php
<?php

namespace Biz;

class Foo
{
    private $baz;

    private function bar()
    {
        $this->baz=5;
    }
}
```

It seems that the sniff also enforces that there is at least 1 space placed before the assignment operator which is indicated by neither the name nor the documentation (in code).

Is this intentional? Altough i personally agree that it should be there, i don't think this sniff should be the one responsible for checking or reporting it.

In the following multiple assignments as well, altough there are no spaces the equals signs are technically aligned.
```php
$this->baz=5;
$this->bar=10;
```

**Comment by gsherwood** [comment]
> Is this intentional?

Yes, the sniff has always worked this way. It's basically a short-cut for ensuring there is correct spacing before the equals sign (1 or more).

There is no sniff that only checks for a single space before the equals sign, although there is `Squiz.WhiteSpace.OperatorSpacing` that checks all operators.

So while I agree that this check could be in another sniff, changing it would be a BC break and require developers to change their rulesets to include it. I'm not sure that is worth doing before a major release.
